src/renderers/dom/client/wrappers/ReactDOMInput.js

```diff
@@ -81,7 +82,7 @@ var ReactDOMInput = {
       onChange: inst._wrapperState.onChange,
     });
 
-    return nativeProps;
+    return DisabledInputUtils.getNativeProps(inst, nativeProps);
```

Comment:
Okay. I believe I've addressed the original desire with 2eeebf9. I've also rebased from upstream.